Libraries/Components/Button.js

```diff
@@ -273,11 +278,14 @@ class Button extends React.Component<ButtonProps> {
         buttonStyles.push({backgroundColor: color});
       }
     }
-    const accessibilityState = {};
+    const accessibilityState =
+      disabled != null
```

Comment:
Alternatively we could do `const accessibilityState = disabled === true ? ... : ...` so it's `accessiblityState.disabled | disabled`.